services/notify/src/exchanges.js

```diff
@@ -28,12 +28,12 @@ const buildCommonRoutingKey = function(options) {
       constant: 'primary',
       required: true,
     }, {
-      name: 'reserved',
-      summary: 'Space reserved for future routing-key entries, you ' +
-                        'should always match this entry with `#`. As ' +
-                        'automatically done by our tooling, if not specified.',
+      name: 'topic',
```

Comment:
I'm not really sure if `topic` is the right word here. It might have a slightly different meaning in RabbitMQ parlance. But I couldn't think of anything better.